effects/freak_effect.py

```diff
@@ -0,0 +1,43 @@
+import message_text_config as msg
+import utils
+from citizens.citizen import Citizen
```

Comment:
please, remove citizen import if it is not necessary for code work.
For type hints you can use strings